### DISCUSSION TITLE
How can I bridge customs messages from ROS to mqtt?

### DISCUSSION THREAD

**Discussion by Blackra1n5** [ika-rwth-aachen/mqtt_client#21] [Q&A]
I have successfully gotten the node to connect to both ROS master and MQTT server, however I am unable to receive a non-jumbled custom message type (example shown below). The message type is 2 values with float32 data type.
Please let me know if I'm missing something!

`
9e237a0971ec09fc952c0717bd077bf2low_level_msgs/BatteryMsg"float32 voltage float32 current 
`

Thanks a ton!

**Comment by lreiher** [comment]
I suggest that you open an issue instead of this discussion. Please also post a copy of your msg definition and the mqtt_client config file that you use.
